config/routes.rb

```diff
@@ -12,5 +12,9 @@
   end
 
   resources :users
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+ 
+  namespace :games do 
+    resources :searches, only: :index, defaults: { format: :json }
+  end
```

Comment:
[IMO]
```suggestion
resources :games, only: [], defaults: { format: :json } do
  get :searches, on: :collection
end
```
とするとgames_controllerにsearchesアクション作るだけで良いかもですね

Reply:
参考までにですが
```suggestion
namespace :api, format: 'json' do
  resources :games, only: :index
end
```
json返すルーティグにapiというnamespaceを切るのは結構やりがちです